<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>databaseInsertion</class>
 <widget class="QDialog" name="databaseInsertion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Site</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name of site:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="nameOfSite">
     <property name="toolTip">
      <string>Name of the site to be added.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Link to site:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="linkToSite">
     <property name="toolTip">
      <string>Link to the Website (e.g. https://example.com/{}) The {} is where the username would be.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="siteIsNSFW">
     <property name="toolTip">
      <string>Check this if the site is for Adult audiences.</string>
     </property>
     <property name="text">
      <string>NSFW Site</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="toolTip">
      <string/>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>databaseInsertion</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>databaseInsertion</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
